### ISSUE TITLE
NullPointerException when adding Git dependency

### ISSUE THREAD

**Issue by NickSeagull** [typelead/gradle-eta#33]
When adding a git dependency to the dependencies, Gradle crashes with a `NullPointerException`.

# Steps to reproduce
1. Clone the `eta-init` project
2. Add the following to the dependencies section:
    ```groovy
    compile eta(package:  'eta-spark-core',
              location: 'https://github.com/Jyothsnasrinivas/eta-spark-core',
              branch:   'master')
    ```
3. Run `./gradlew run`
4. Crash

# Additional Information
Tried on **Ubuntu 16.04 (WSL)**, and **Windows 10 x64**.

Tried with other packages like `dhall-eta` and `eta-prelude`.

# [StackTrace](https://gist.github.com/NickSeagull/ef17efd0f3737900a78d134a11632f39)

**Comment by rahulmutt** [comment]
Thanks for reporting! Was able to reproduce. I'll add a git dependency example to the test suite so we catch this sort of regression in the future.

**Comment by rahulmutt** [comment]
Fixed in `0.8.0`

**Comment by NickSeagull** [comment]
Great, thanks!!
